test/models/value_definition_test.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ValueDefinitionTest < ActiveSupport::TestCase
+  should belong_to(:benchmark)
+  should validate_presence_of :ref_id
+  should validate_uniqueness_of(:ref_id).scoped_to(:benchmark_id)
+  should validate_presence_of(:value_type)
+  should validate_presence_of(:description)
```

Comment:
Maybe a test case where you try to assign something else to the value type than allowed?

Reply:
^^^